ipynb/filterEMP.ipynb

```diff
@@ -0,0 +1,254 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Author**: stefan.m.janssen@gmail.com<br>\n",
+    "**Date**: 08 Aug 2016<br>\n",
+    "**language**: Python 3.5.2 :: Continuum Analytics, Inc.<br>\n",
+    "**conda enviroment**: micronota<br>\n",
+    "**license**: unlicensed<br>\n",
+    "\n",
+    "# Filter EMP\n",
+    "Currently the EMP comprises ~34k samples. Sometimes we need to operate on smaller sample numbers. Thus, the merged mapping file contains 'subset_xxx' columns, where xxx indicates the number of samples in the subset. These columns are np.boolean and flags if a sample of the whole EMP is part of subset xxx.\n",
+    "\n",
+    "There are several point where we need to filter full data-sets to those smaller subsets, e.g. BIOM tables or beta diversity distance matrices. This notebook is an example of how to acomplish that. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "# a trick to make it transparent if executed on a barnacle node or on my machine with barnacles file system mounted to /media/barnacle\n",
+    "import socket\n",
+    "ROOT = \"/media/barnacle/\" if socket.gethostname() == 't440s' else \"/\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "%matplotlib inline\n",
+    "\n",
+    "# make all necessary imports\n",
+    "import pandas as pd\n",
+    "import numpy as np\n",
+    "import sys\n",
+    "import seaborn as sn\n",
+    "import matplotlib.pyplot as plt\n",
+    "import os\n",
```

Comment:
you're not using `sys`, `seaborn`, `plt` or `os` in this notebook. No need to import them